pkg/pillar/zedcloud/send.go

```diff
@@ -281,8 +294,24 @@ func SendOnIntf(ctx ZedCloudContext, destUrl string, intf string, reqlen int64,
 		}
 
 		var req *http.Request
-		if b != nil {
-			req, err = http.NewRequest("POST", reqUrl, b)
+		var b2 *bytes.Buffer
+		if ctx.V2API && isEdgenode && !ishttpGet {
+			b2, err = addAuthentication(ctx, b, useOnboard)
+			if err != nil {
+				log.Infof("SendOnIntf: auth error %v\n", err)
+				return nil, nil, remExtStatus, err
+			}
+			log.Infof("SendOnIntf: add auth for %s\n", reqUrl) // XXX move to debug later
+		} else {
+			b2 = b
+		}
+
+		if b2 != nil {
+			if ishttpGet {
+				req, err = http.NewRequest("GET", reqUrl, b2)
```

Comment:
So you are sending a payload with the GET? What does the controller do with that?

Reply:
maybe i don't need this. i thought we had a case of that, but can not find it. will remove this.